plugin/pick.vim

```diff
@@ -56,7 +60,7 @@ function! s:FileListCommand()
 endfunction
 
 function! s:GitListCommand(file)
-  return "git ls-files " . a:file . " --cached --exclude-standard --others"
```

Comment:
Since this affects the behavior of `PickFile` and friends, I'm trying to figure out what would be the most intuitive/helpful behavior of those functions.

Personally, I'd expect these functions to let me pick from all files in the working directory.

Does that make sense? If so, I'd rather keep the current behavior given the possibility to call `PickCommand` directly;.